### DISCUSSION TITLE
App still on v055 in F-Droid

### DISCUSSION THREAD

**Discussion by steinland1994** [celzero/rethink-app#1046] [Q&A] [answered]
Hi,

v055a is still not available to me through fdroid.

Normally I would just assume this is intended, but the build status is shown as successful and [this file](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.celzero.bravedns.yml) says v055a is the current version that's supposed to be on the fdroid store.

Is there something else holding up the rollout?

lg

A

**Comment by retroluv** [comment, marked as answer]
Hi, it has been available since sometime yesterday.

https://f-droid.org/en/packages/com.celzero.bravedns/
